src/backbone-marionette.js

```diff
@@ -0,0 +1,65 @@
+import
+  {
+    bindEntityEvents,
+    unbindEntityEvents,
+    proxyBindEntityEvents,
+    proxyUnbindEntityEvents
+  } from './bind-entity-events';
+
+import extend             from './utils/extend';
+import isNodeAttached     from './utils/isNodeAttached';
+import mergeOptions       from './utils/mergeOptions';
+import getOption          from './utils/getOption';
+import proxyGetOption     from './utils/proxyGetOption';
+import _getValue          from './utils/_getValue';
+import normalizeMethods   from './utils/normalizeMethods';
+import normalizeUIString  from './utils/normalizeUIString';
+import normalizeUIKeys    from './utils/normalizeUIKeys';
+import normalizeUIValues  from './utils/normalizeUIValues';
+import actAsCollection    from './utils/actAsCollection';
+import deprecate          from './utils/deprecate';
+
+import MonitorDOMRefresh from './dom-refresh';
+import
+  {
+    _triggerMethod,
+    triggerMethod,
+    triggerMethodOn,
+    triggerMethodMany
+  } from './trigger-method';
+
+import { FEATURES, isEnabled } from './features';
+
+import MarionetteError from './error';
+
+Marionette = Marionette || {};
+
+// extend Marionette in the meantime
+Object.assign(Marionette, {
```

Comment:
I wonder if everything still needs to be attached to `Mn`?  Seems like a few things were attached to the root simply because we weren't using module loaders and couldn't import them.  Like specifically if we're not calling `Marionette._getValue()` anywhere we probably don't need to attach it..  I suspect maybe the proxy functions as well?